### DISCUSSION TITLE
Script update nuaces

### DISCUSSION THREAD

**Discussion by 7nik** [Tampermonkey/tampermonkey#1542] [Q&A] [answered]
Hi, @derjanb, I'm going to rewrite one of my usercripts and move it to another repo so I have a few questions about how the script will update:
1. Does the `@downloadURL` have higher priority than the URL of installation (when the script was installed by a link)?
2. What will happen if the `@downloadURL` points to a script with technically another name?

**Comment by derjanb** [comment, marked as answer]
> Does the `@downloadURL` have higher priority than the URL of installation (when the script was installed by a link)?

Yes.

> What will happen if the `@downloadURL` points to a script with technically another name

Tampermonkey will update from that URL and rename the script accordingly.
When the final update URL changes, a script update confirmation dialog is shown by Tampermonkey to confirm the new script update source.
